### PR TITLE
Present information in an easy-to-digest format #18

### PR DESCRIPTION
**Pull request by zkrzyzanowski**
Addresses issue #18 
* Console logging is  formatted as ` [impact]: [help]. Offending markup is: [nodes.html]`
* Ex: `critical: Form elements must have labels. Offending markup is: <input id="test" type="test">`

### CODE REVIEW

File: tests/integration/instance-initializers/axe-component-test.js [hunk 1]
```diff
@@ -122,7 +122,14 @@ test('audit should log any violations found', function(assert) {
   stubA11yCheck(sandbox, {
     violations: [{
       name: 'test',
-      nodes: []
+      nodes: [
```
Comment (trentmwillis) [inline tests/integration/instance-initializers/axe-component-test.js]: Can we add a test where there are no `nodes`? That will account for the `if (!nodes) {` branch added to the code above.

**Comment by trentmwillis** [comment]
Here is a before/after comparison of the changes in action.

Logging before:
screen shot 2017-04-27 at 1 12 20 pm

Logging after (much more clear):
screen shot 2017-04-27 at 1 12 03 pm

File: tests/integration/instance-initializers/axe-component-test.js (hunk 1 above)
Comment (zkrzyzanowski) [inline tests/integration/instance-initializers/axe-component-test.js]: @trentmwillis Can you take a look at my updates on this PR?

**Comment by trentmwillis** [comment]
Released in 0.4.3!

**Pull request merged**